src/components/common/copyable-button/copyable-button.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Fragment, useCallback, useRef, useState } from 'react'
+import { Button, Overlay, Tooltip } from 'react-bootstrap'
+import { Variant } from 'react-bootstrap/types'
+import { Trans, useTranslation } from 'react-i18next'
+import { ForkAwesomeIcon } from '../fork-awesome/fork-awesome-icon'
+
+export interface CopyableButtonProps {
+  content: string
+  size?: 'sm' | 'lg'
+  variant?: Variant
+}
+
+export const CopyableButton: React.FC<CopyableButtonProps> = ({ content, size = 'sm', variant = 'dark' }) => {
```

Comment:
The button is copyable?

Reply:
You copied (no pun inteded) the logic from the version info modal, right? Why don't you use the button there?

Reply:
I think it's better to show the URL there. In the Code the content is already visible.

Reply:
Nevertheless "CopyableButton" is a weird name, because it implies that this button is copyable. (Yes I know that is adapted from CopyableField, but that's a strange name, too). I would use something like "WriteToClipboardButton" or something.

Reply:
CopyButton?

Reply:
CopyToClipboardButton? I would include the clipboard to be more specific.

Reply:
Not to sure, but if you insist…

Reply:
done